frontend/src/apiAccess/apis.js

```diff
@@ -379,6 +379,40 @@ async function activityLogDayCountsPage({
   return { response, responseData };
 }
 
+async function activityLogEntries({
```

Comment:
should the function name not also be updated? at the moment it seems to be missing a verb to show what the function is doing with the activity log entries.

Reply:
i think the function name is okay, looking at the entire structure of the file